static/src/javascripts/projects/common/modules/gallery/lightbox.js

```diff
@@ -295,16 +297,97 @@ class GalleryLightbox {
         this.fsm.trigger(event, data);
     }
 
-    loadGalleryfromJson(galleryJson: GalleryJson, startIndex: number): void {
-        this.index = startIndex;
+    static loadNextOrPrevious(
+        currentImageId: string,
+        direction: string
+    ): Promise<Object> {
+        const pathPrefix = direction === 'forwards' ? 'getnext' : 'getprev';
+        const seriesTag = config
+            .get('page.nonKeywordTagIds')
```

Comment:
Could this ever be null or undefined? Might be worth specifying a default of `''` (empty string), as the second parameter to this function call.